### DISCUSSION TITLE
Wishlist

### DISCUSSION THREAD

**Discussion by ralfHielscher** [mtex-toolbox/mtex#518] [Feature Requests]
Hi MTEX users,

we are eager to hear your ideas how to improve and extend MTEX. This thread is meant as an overview over all new ideas, wishes and features requests. 

You are welcome to **vote** for a feature by clicking the arrows at the left hand side.

Discussions on existing feature request should **not** take place in this wish-list but in the linked issues / discussion thread of each idea.

You are welcome to **add** a feature request to this list. Please do this as follows:

1. Open an [issue](https://github.com/mtex-toolbox/mtex/issues/new?assignees=&labels=feature+request+%3Achristmas_tree%3A&template=feature_request.md&title=) or a [discussion](https://github.com/mtex-toolbox/mtex/discussions/new?category_id=32000417) where you describe the requested feature in detail. This includes use case, possible syntax, links to definitions and formulae. 
2. Add a post to this wish-list where you shortly describe the requested feature and provide a link to the issue / discussion created in 1.

Discussions or doubled requests will be deleted to keep the list comprehensive.

Your MTEX team.

**Comment by ralfHielscher** [comment]
Allow to plot internal faces in crystal shape plots #504

**Comment by ralfHielscher** [comment]
Align spherical plots with z-Axis not perpendicular to the plane #520

**Comment by jhiscocks** [comment]
More consistency with in Miller index definition with regards to hkil (must be lower case) vs UVTW  (must be upper case) and corresponding documentation. #530

**Reply by ralfHielscher** [reply]
Hi Jessica,

I created a feature #530 request for your wish. Could you please add more details there?

Ralf.

**Comment by oMuransky** [comment]
Implement an MTEX interface to [Neper](https://neper.info/) or [Dream3d](https://github.com/BlueQuartzSoftware/DREAM3D) for the generation of an RVE (Representiv Volume Element). More details at #531.

**Comment by ThomasChauve** [comment]
Hello
Thanks for the great MTEX toolbox. I have been using it for years.
If I may have a wish, it wiil be to have MTEX Toolbox in Python. I love jupyter notebook and I think it will be easier to link with other scientific code.
Best
Thomas

**Comment by bjoerneske** [comment]
Make the micronbar easier to customize. Possible properties would be: position, size, fontsize, ....
More details in #836.

**Comment by ralfHielscher** [comment]
Implement a class twinSystem similar to slipSystem. See #870

**Comment by ralfHielscher** [comment]
Twin Reconstruction see #871

**Comment by ralfHielscher** [comment]
Fabric coordinate system and fabric orientations for structural geologists. See #872

**Comment by ralfHielscher** [comment]
Deformation / Kinematic Reference frame #873

**Comment by ralfHielscher** [comment]
Pseudo 3d EBSD data #875
